engine/src/main/resources/org/camunda/bpm/engine/db/liquibase/camunda-changelog.xml

```diff
@@ -117,9 +117,16 @@
              stripComments="true"/>
   </changeSet>
 
-  <changeSet author="Camunda" id="7.20.0-tag">
-    <tagDatabase tag="7.20.0"/>
```

Comment:
🔧 I believe the `7.20` tag still needs to remain, doesn't it?

Reply:
👍